scripts/completion-tests/completionTests.sh

```diff
@@ -0,0 +1,64 @@
+#!bash
+#
+# Copyright (C) 2019 Ville de Montreal
```

Comment:
The copyright should read:

> Copyright The Helm Authors.

Can you please update it throughout the PR.

Reply:
I was hoping you'd catch this since I wasn't sure how to handle it.

IIUC the DCO is a way for me to contribute this code to the project under the licence used by the project; the DCO does not transfer the copyright ownership of the code however. 

So, since I'm not a Helm Author and don't have authorization to give away the copyright, I didn't think it was ok to put the "The Helm Authors".  In other projects I've seen the form "<original copyright holder> and others" to more appropriately reflect the copyright ownership.

Do you know of guidelines from the CNCF on this?

I don't mean to be a pain, but I also don't want to overstep what I'm allowed to do.

Reply:
The CNCF wants all projects to be Apache 2 licensed. One of the reasons is because of point 5 under the terms of use, which calls this out explicitly: https://github.com/helm/helm/blob/master/LICENSE#L131-L137

Reply:
This contribution is made under the Apache 2 license, so it can be part of the project directly.

It's just that the Copyright ownership is not transferred by the DCO, so I thought it should be written properly who the copyright owner is, in this case Ville de Montreal.  But the license remains Apache 2.

Reply:
To complete on this discussion, I've contacted the CNCF about the copyright question and they explained that by making a contribution to the Helm project, my employer (through me) becomes a "Helm Author". Therefore, it is ok to put "Copyright The Helm Authors" as part of my contribution even though the copyright is not transferred. Sorry for me not realizing this.